test/test_logs.sh

```
@@ -0,0 +1,365 @@
+GIT_EOF
+  chmod +x "$mock_dir/git"
+}
+
+setup_mocks_logs() {
+  local check_runs="$1"
+  local log_content="$2"
+
+  setup_mocks
+
+  printf '#!/usr/bin/env bash\ncase "$*" in\n' > "$mock_dir/gh"
+  printf '  *"pulls/42"*)\n' >> "$mock_dir/gh"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)